src/components/product.js

```diff
@@ -195,13 +188,11 @@ export default class Product extends Component {
 
   get carouselImages() {
     return this.model.images.map((image) => {
-      return Object.assign({}, image, {
-        // isSelected: image.id === this.currentImage.id,
-        isSelected: true,
-        // use correct image size
-        // carouselSrc: image.variants[3].src,
```

Comment:
Awesome - we'll have to remember to clean up cruft once we've got everything sorted.